scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -58,10 +54,31 @@ workflow Mutect2 {
   String? sequencing_center
   String? sequence_source
   File? default_config_file
+  File? tumor_sequencing_artifact_metrics
   Boolean is_bamOut = false
 
   # Do not populate unless you know what you are doing...
   File? auth
+  # This value is added to every tasks disk in case the dynamic sizing isnt enough
```

Comment:
`isn't` 

Also, you should check that DSP Comms do their WDL docs like this.

Reply:
what do you mean by "like this" ?